doc/src/region.rst

```diff
@@ -225,6 +226,21 @@ keywords for the simulation box parameters and timestep and elapsed
 time.  Thus it is easy to specify a time-dependent radius or have
 a time dependent position of the sphere or cylinder region.
 
+.. note::
+
+   Whenever a region property, such as a coordinate or an upper/lower
+   bound, is defined via an equal-style variable, it must be ensured
```

Comment:
... via an equal-style variable, the variable should not cause any of the region boundaries ...